### PR TITLE
feat: add header and footer

### PR DESCRIPTION
**Pull request by adriansberg**
Added header and footer to the base layout. Reused the components from the online store and removed most of the irrelevant features that is not applicable to the travel planner.

<details><summary>Screenshots</summary>
<p>
Full page:
image1

</p>
</details>

### CODE REVIEW

**Review by mikaelbr — APPROVED** [review]
Should make a issue to make environments configurable

File: src/layouts/shared/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+        <section className={style.footer__bottomLinks}>
+          <div className={style.footer__some}>
+            {facebookLink && (
+              <ButtonLink
+                mode={isDarkMode ? 'interactive_2' : 'interactive_1'}
+                radius="top-bottom"
+                display="inline"
+                radiusSize="circular"
+                icon={{ left: <img src="/fb.svg" alt="Facebook" /> }}
```
Comment (mortennordseth) [inline src/layouts/shared/footer.tsx:144]: Looks like these social media icons are missing

Comment (adriansberg) [reply src/layouts/shared/footer.tsx:144]: True! Do we want them so I should add them now? Probably?

File: src/layouts/shared/page-header.module.css [hunk 2]
```diff
@@ -0,0 +1,273 @@
+.pageHeader {
+  background-color: var(--static-background-background_0-background);
+}
+
+.pageHeader__content {
+  width: 100%;
+  max-width: var(--maxPageWidth);
+  margin: 0 auto;
```
Comment (mortennordseth) [inline src/layouts/shared/page-header.module.css:8]: Looking in Figma I think this should be `margin: 0`

Comment (adriansberg) [reply src/layouts/shared/page-header.module.css:8]: You sure? The sketches don't show many examples of wide screens.

File: src/layouts/shared/footer.tsx (hunk 1 above)
Comment (mortennordseth) [reply src/layouts/shared/footer.tsx:144]: They are at least in the Figma sketches, so my guess is that we should have them. If not, we should probably remove the links altogether 🤔

Comment (adriansberg) [reply src/layouts/shared/footer.tsx:144]: I can add them 👍

Comment (mortennordseth) [reply src/layouts/shared/footer.tsx:144]: It seems like it's copy paste from the webshop though (we are upgrading link etc.), so not sure how thought-through it is 🤔

Comment (adriansberg) [reply src/layouts/shared/footer.tsx:144]: Faster to remove at a later point than add them now that it is being worked on.

Comment (mortennordseth) [reply src/layouts/shared/footer.tsx:144]: True 👍

File: src/layouts/shared/page-header.module.css (hunk 2 above)
Comment (mortennordseth) [reply src/layouts/shared/page-header.module.css:8]: It looks so in this frame 🤔 

image

Comment (adriansberg) [reply src/layouts/shared/page-header.module.css:8]: Let's have a design review meeting at the end of this week to clean up stuff like this. Leaving it as is for now to keep it similar to the webshop's.

**Review by mortennordseth — APPROVED** [review]

**Pull request merged**